request_test.go

```diff
@@ -26,7 +26,12 @@ func TestCreateRequest(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		req := CreateRequest(tc.url, tc.method)
+		req := &Request{
```

Comment:
Please delete the `TestCreateRequest` test as it's not usefull anymore.

Reply:
done